fluentd.gemspec

```diff
@@ -50,7 +50,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("parallel_tests", ["~> 0.15.3"])
   gem.add_development_dependency("simplecov", ["~> 0.7"])
   gem.add_development_dependency("rr", ["~> 3.0"])
-  gem.add_development_dependency("timecop", ["~> 0.9"])
+  gem.add_development_dependency("timecop", ["< 0.9.9"])
```

Comment:
Could you leave a brief comment "why" we need specific version and how to resolve in the future.

(though we can see git blame and git log)

Reply:
Added!